presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -932,7 +932,7 @@ public void createTable(ConnectorSession session, ConnectorTableMetadata tableMe
             }
 
             tableType = EXTERNAL_TABLE;
-            targetPath = getExternalPath(new HdfsContext(session, schemaName, tableName), externalLocation);
+            targetPath = getExternalPath(new HdfsContext(session, schemaName, tableName, true), externalLocation);
```

Comment:
Let's provide the table location here as well, it should be the `externalLocation `